Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3-alpine
+
+WORKDIR /action
+
+# Copy action metadata
+COPY LICENSE README.md requirements.txt ./
```

Comment:
don't think you need LICENSE and README.md in the docker container

Reply:
I think you're right. Not sure how this evolved, probably came from an example I was following at some point lol. Will remove 👍

Reply:
Done in https://github.com/dabrady/syndicate/pull/1/commits/54c83875e4607667c7b90877617f195e87c2916f